cmd/atlascmd/schema.go

```diff
@@ -167,14 +170,16 @@ func CmdInspectRun(cmd *cobra.Command, _ []string) {
 
 // CmdApplyRun is the command used when running CLI.
 func CmdApplyRun(cmd *cobra.Command, _ []string) {
+	args, err := argMap(ApplyFlags.Vars)
+	cobra.CheckErr(err)
```

Comment:
See my other comment, this is obsolete then and you can pass in the flag directly to `applyRun`.